ocpmodels/common/utils.py

```diff
@@ -545,11 +545,24 @@ def get_pbc_distances(
 
 
 def radius_graph_pbc(
-    data, radius, max_num_neighbors_threshold, pbc=[True, True, False]
+    data, radius, max_num_neighbors_threshold, pbc=[True, True, True]
```

Comment:
@mshuaibii @AdeeshKolluru do you see any issues making this the default for our models (when `otf_graph=True`)?

Reply:
I think that's fine.